classes/class-twenty-twenty-one-custom-colors.php

```diff
@@ -115,8 +130,12 @@ public function editor_custom_color_variables() {
 			array(),
 			(string) filemtime( get_theme_file_path( 'assets/css/custom-color-overrides.css' ) )
 		);
+
 		if ( 'd1e4dd' !== strtolower( get_theme_mod( 'background_color', 'D1E4DD' ) ) ) {
 			wp_add_inline_style( 'twenty-twenty-one-custom-color-overrides', $this->generate_custom_color_variables( 'editor' ) );
+		} elseif ( $this->is_default_palette() ) {
+			// Add dark mode variable overrides.
+			wp_add_inline_style( 'twenty-twenty-one-custom-color-overrides', '@media (prefers-color-scheme: dark) { :root .editor-styles-wrapper { --global--color-background: var(--global--color-dark-gray); --global--color-primary: var(--global--color-light-gray); --global--color-secondary: var(--global--color-light-gray); } }' );
 		}
```

Comment:
The first conditional is catching when a different light background is chosen (ex, #e4dad1), so the editor shows the light background colors instead.

I think you want two separate conditionals to merge the two CSS outputs, so that the custom color overrides can be loaded in with the dark-mode  media query.

Reply:
Fixed, thank you!